playbook/CHANGELOG.md

```diff
@@ -222,3734 +222,3 @@ A new Quick Pick variant now in DatePicker! This sleek and intuitive component p
 [Full Changelog](https://github.com/powerhome/playbook/compare/12.28.0...12.29.0)
 
 # Updated Phone Number Input: Easier Value Manipulation in React Forms
```

Comment:
The changelog ends abruptly. Take a look and lets end it after the newer posts with images.  
<img width="1065" alt="image" src="https://github.com/powerhome/playbook/assets/863031/6dde3e2d-2078-4491-a56e-214437645350">